cfgov/jinja2/v1/rural-or-underserved/index.html

```diff
@@ -294,20 +294,24 @@ <h2 class="js-table-title">
                             First-lien covered transactions extended in <span class="chosenYear">2015</span> that are secured by these properties can be counted as rural or underserved in your determination of your first-lien covered transactions in <span class="chosenYear">2015</span> that are secured by properties located in rural or underserved areas.
                         </p>
 
-                        <table class="table" style="border-width: 1px;" width="100%">
-                            <thead>
-                                <tr>
-                                <th width="31.5%">Address Entered</th>
-                                <th width="31.5%">Address Identified</th>
-                                <th width="15%">County</th>
-                                <th width="22%">Rural or Underserved?</th>
-                                </tr>
-                            </thead>
-                            <tbody class="data"></tbody>
-                        </table>
-
-                        <a data-table="rural" class="button-more u-hidden a-btn a-btn__secondary no-decoration" href="#" id="ruralMore">View 10 more</a>
-                        <a data-table="rural" class="margin-left-1 view-all u-hidden no-visit" href="#" id="ruralAll">View all</a>
+                        <div class="o-table o-table-wrapper__scrolling">
+                            <table>
+                                <thead>
+                                    <tr>
+                                        <th width="31.5%">Address Entered</th>
+                                        <th width="31.5%">Address Identified</th>
+                                        <th width="15%">County</th>
+                                        <th width="22%">Rural or Underserved?</th>
+                                    </tr>
+                                </thead>
+                                <tbody class="data"></tbody>
+                            </table>
+                        </div>
+
+                        <p class="block block__flush-top block__flush-bottom block__padded-top block__padded-bottom">
+                            <a data-table="rural" class="button-more u-hidden a-btn a-btn__secondary no-decoration" href="#" id="ruralMore">View 10 more</a>
+                            <a data-table="rural" class="margin-left-1 view-all u-hidden no-visit" href="#" id="ruralAll">View all</a>
```

Comment:
Let's give these buttons the same separation as similar button combos on cf.gov, like the "Apply filters" and "Clear filters" buttons in filter modules. I think that's a 5px separation.

Reply:
It'll be updated to this in the next push
<img width="251" alt="Screen Shot 2019-07-16 at 10 18 24 AM" src="https://user-images.githubusercontent.com/704760/61302026-1aac6100-a7b3-11e9-9b0b-ea890dbed15b.png">

Reply:
Also opened https://github.com/cfpb/capital-framework/issues/946

Reply:
These should all be `button` elements, too.

Reply:
Ah, true

Reply:
The spacing looks good to me